apps/web/src/services/cost-explorer/cost-analysis/config.ts

```diff
@@ -0,0 +1,51 @@
+import dayjs from 'dayjs';
+
+import { GRANULARITY, GROUP_BY } from '@/services/cost-explorer/lib/config';
+import type { CostQuerySetModel, Period } from '@/services/cost-explorer/type';
+
+export const MANAGED_COST_QUERY_SET_IDS = {
+    MONTHLY_PROJECT: 'Monthly cost by project',
+    MONTHLY_SERVICE_ACCOUNT: 'Monthly cost by service account',
+    MONTHLY_PRODUCT: 'Monthly cost by product',
+} as const;
+
+export const managedCostQuertSetIdList: string[] = [
+    MANAGED_COST_QUERY_SET_IDS.MONTHLY_PROJECT,
+    MANAGED_COST_QUERY_SET_IDS.MONTHLY_SERVICE_ACCOUNT,
+    MANAGED_COST_QUERY_SET_IDS.MONTHLY_PRODUCT,
+];
+
+const defaultPeriod: Period = {
+    start: dayjs.utc().subtract(5, 'month').startOf('month').format(),
+    end: dayjs.utc().endOf('month').format(),
+};
```

Comment:
nitpick: there's `getInitialDates()` in cost-explorer/lib/helper.ts !

Reply:
Thanks!